test/benchmarks/mdc/menu/menu.html

```diff
@@ -0,0 +1,33 @@
+<button mat-button [matMenuTriggerFor]="basic">Basic Menu</button>
+<button mat-button [matMenuTriggerFor]="nested">Nested Menu</button>
+
+<mat-menu #basic="matMenu">
+	<button mat-menu-item>Item 1</button>
+	<button mat-menu-item>Item 2</button>
+	<button mat-menu-item>Item 3</button>
+	<button mat-menu-item>Item 4</button>
+	<button mat-menu-item>Item 5</button>
```

Comment:
Looks like this file is using tabs for indentation, whereas the rest of the project is using spaces.